SPECS/azurelinux-repos/azurelinux-amd-preview.repo

```diff
@@ -0,0 +1,19 @@
+[azurelinux-official-amd-preview]
```

Comment:
The toolchain manifest made me realize there's no `*-debug` version of these new repos. Is that by design? If we're not publishing any symbols for the RPMs, then that would make sense. Otherwise, please also add the `*-debug` counterparts. Debugging may be trickier without them.;)

Reply:
yes, there's no debug symbols for this one